android/build.gradle

```diff
@@ -58,9 +59,11 @@ dependencies {
     embed project(':core')
     embed project(':event-bridge')
     embed project(':id')
+    compileOnly project(':android-compose')
```

Comment:
I haven't seen `compileOnly` used on a subproject before. Does this selectively include the subproject source code? 

I was expecting you would have the compose libraries here instead (which could be added by the customers app)
```
    compileOnly(compose.runtime)
    compileOnly(compose.ui)
```

Reply:
I just realized that I misunderstood what compileOnly actually does! https://github.com/amplitude/Amplitude-Kotlin/pull/199/commits/ffa1aa1349bd618d7e04cafafeff892fec35fc32 should fix it now.